### PR TITLE
Removed use of mocha from railties actions_test

### PR DESCRIPTION
**Pull request by ronakjangir47**

### CODE REVIEW

File: railties/test/generators/actions_test.rb [hunk 1]
```diff
@@ -140,13 +142,18 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
   end
 
   def test_git_with_symbol_should_run_command_using_git_scm
-    generator.expects(:run).once.with('git init')
-    action :git, :init
+    @mock_generator.expect(:call,nil,['git init'])
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 2]
```diff
@@ -140,13 +142,18 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
...
+    generator.stub(:run,@mock_generator) do
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 3]
```diff
@@ -140,13 +142,18 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
...
+      action :git, :init
+    end
   end
 
   def test_git_with_hash_should_run_each_command_using_git_scm
-    generator.expects(:run).times(2)
-    action :git, rm: 'README', add: '.'
+    @mock_generator.expect(:call,nil,["git rm README"])
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 4]
```diff
@@ -140,13 +142,18 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
...
+    @mock_generator.expect(:call,nil,["git add ."])
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 5]
```diff
@@ -140,13 +142,18 @@ def test_environment_with_block_should_include_block_contents_in_environment_ini
...
+    generator.stub(:run,@mock_generator) do
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 6]
```diff
@@ -170,46 +177,60 @@ def test_initializer_should_write_date_to_file_in_config_initializers
   end
 
   def test_generate_should_run_script_generate_with_argument_and_options
-    generator.expects(:run_ruby_script).once.with('bin/rails generate model MyModel', verbose: false)
-    action :generate, 'model', 'MyModel'
+    @mock_generator.expect(:call,nil,['bin/rails generate model MyModel', verbose: false])
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 7]
```diff
@@ -170,46 +177,60 @@ def test_initializer_should_write_date_to_file_in_config_initializers
...
+    generator.stub(:run_ruby_script,@mock_generator) do
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 8]
```diff
@@ -170,46 +177,60 @@ def test_initializer_should_write_date_to_file_in_config_initializers
...
+      action :generate, 'model', 'MyModel'
+    end
   end
 
   def test_rake_should_run_rake_command_with_default_env
-    generator.expects(:run).once.with("rake log:clear RAILS_ENV=development", verbose: false)
-    with_rails_env nil do
-      action :rake, 'log:clear'
+    @mock_generator.expect(:call,nil,["rake log:clear RAILS_ENV=development", verbose: false])
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

File: railties/test/generators/actions_test.rb [hunk 9]
```diff
@@ -170,46 +177,60 @@ def test_initializer_should_write_date_to_file_in_config_initializers
...
+    generator.stub(:run,@mock_generator) do
```
Comment (rafaelfranca) [inline railties/test/generators/actions_test.rb]: space after the `,`

**Comment by ronakjangir47** [comment]
@rafaelfranca Done with changes, please let me know if anything!

**Comment by ronakjangir47** [comment]
Thanks @rafaelfranca

**Pull request merged**